zk.go

```diff
@@ -14,7 +14,7 @@ import (
 )
 
 const (
-	DefaultTimezone = "Asia/Ho_Chi_Minh"
+	DefaultTimezone = "Europe/Paris"
```

Comment:
@willylambert do not change the default value. If you want to change the timezone for testing. Please create new test case in the test file.